commons/src/main/resources/kapua-environment-setting.properties

```diff
@@ -120,6 +120,10 @@ commons.cache.provider.classname=org.eclipse.kapua.commons.service.internal.cach
 #
 commons.cache.local.tmetadata.maxsize=100
 
+commons.authentication-cache.config.size=1000
```

Comment:
This settings are authentication settings and are betters suit to live in the authentication module. 

Actually we have already a `authentication.credential.userpass.cache.ttl` property in the `kapua-authentication-setting.properties` setting file which we are not using. We might re-use that and add a new `authentication.credential.userpass.cache.size`